test/snippets/react-context-value-func.tsx

```diff
@@ -10,7 +10,7 @@ const ContextProvider = ({ children }) => {
   return <Context.Provider value={cb}>{children(foo)}</Context.Provider>;
 };
 
```

Comment:
this snippets folder shouldn't be linted. We don't actively run it in CI, we could potentially delete it but it was previously "weird code" that was useful to run sometimes